src/app/donation-start/donation-start-form/donation-start-form-new.component.html

```diff
@@ -521,45 +522,65 @@ <h3 class="b-rt-0 b-m-0 b-bold">How does Big Give use tips?</h3>
             {{ stripeError }}
           </p>
 
+          <p></p>
+
+          <table class="summary" *ngIf="donation">
+            <tr class="donationAmount">
+              <td>Your donation to {{ campaign.charity.name }}</td>
+              <td>{{ donationAmount | exactCurrency:donation.currencyCode }}</td>
+            </tr>
+            <tr *ngIf="donation.matchedAmount > 0">
+              <td>Matched Amount</td>
+              <td>{{ expectedMatchAmount() | exactCurrency:donation.currencyCode }}</td>
+            </tr>
+            <tr *ngIf="giftAidAmount() > 0">
+              <td>Gift Aid Amount</td>
+              <td>{{ giftAidAmount() | exactCurrency:donation.currencyCode }}</td>
+            </tr>
+            <tr *ngIf="feeCoverAmount() > 0">
+              <td>Platform fee</td>
+              <td>{{ feeCoverAmount() | exactCurrency:campaign.currencyCode }} </td>
+            </tr>
+
+            <!-- If the donor is using credit we wouldn't be taking a tip at this point,
+                   and if we are charging a campaign fee we wouldn't have asked for a tip
+                   -->
+            <tr
+              *ngIf="creditPenceToUse === 0 && !campaign.feePercentage"
+            >
+              <td>Tip Amount</td>
+              <td>{{ donation.tipAmount | exactCurrency:donation.currencyCode }}</td>
+            </tr>
+
+            <tr>
+              <td colspan="2"><hr></td>
+            </tr>
+            <!-- TODO: think about to include matchfunding in the total -->
```

Comment:
I don't think I agree with this _todo_, as IMO it is very likely we would get donors nervous about being overcharged.

Reply:
I think I put this in because I was looking at the thanks page, idea was to align the two better. Thinking now maybe we'd even want them to share code if we want them really aligned. Not sure.

This is the thanks page we have now:

![image](https://github.com/thebiggive/donate-frontend/assets/159481/28d4965e-426b-47f2-8bd3-2b92de176282)